File: uniapp_flutter/lib/validations/client_validation_response.dart
class ClientValidationResponse {
  bool mIsValid;
  String mMessage;
  Map<String, String> mKeyToErrorMessage;

  ClientValidationResponse ok() {
    ClientValidationResponse clientValidationResponse = new ClientValidationResponse();
    clientValidationResponse.mIsValid = true;
    clientValidationResponse.mKeyToErrorMessage = new Map();
    return clientValidationResponse;
  }
}

File: app/app/src/main/java/com/vassar/unifiedapp/model/ClientValidationResponse.java
package com.vassar.unifiedapp.model;

import java.util.LinkedHashMap;

public class ClientValidationResponse {

    public boolean mIsValid;
    public String mMessage;
    public LinkedHashMap<String, String> mKeyToErrorMessage;

    public ClientValidationResponse ok() {
        ClientValidationResponse clientValidationResponse = new ClientValidationResponse();
        clientValidationResponse.mIsValid = true;
        clientValidationResponse.mKeyToErrorMessage = new LinkedHashMap<>();
        return clientValidationResponse;
    }
}
